src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/GenerateResponseSealedSpecTest.java

```diff
@@ -0,0 +1,267 @@
+                              .documentBinaryUrl("binary-url")
+                              .build())
+            .build();
+
+    private static final CaseDocument STITCHED_DOC =
+        CaseDocument.builder()
+            .createdBy("John")
+            .documentName("Stitched document")
+            .documentSize(0L)
+            .documentType(SEALED_CLAIM)
+            .createdDatetime(LocalDateTime.now())
+            .documentLink(Document.builder()
+                              .documentUrl("fake-url")
+                              .documentFileName("file-name")
+                              .documentBinaryUrl("binary-url")
+                              .build())
+            .build();
+
+    @BeforeEach
+    void setup() {
+        when(sealedClaimResponseFormGeneratorForSpec.generate(any(CaseData.class), anyString())).thenReturn(SEALED_FORM);
+        when(civilDocumentStitchingService.bundle(ArgumentMatchers.anyList(), anyString(), anyString(), anyString(),
+                                                  any(CaseData.class))).thenReturn(STITCHED_DOC);
+        List<Element<CaseDocument>> systemGeneratedCaseDocuments = new ArrayList<>();
+        systemGeneratedCaseDocuments.add(element(DIRECTIONS_QUESTIONNAIRE_DOC));
+        caseData = CaseDataBuilder.builder()
+            .atStatePendingClaimIssued().build().toBuilder()
+            .specRespondent1Represented(YES)
+            .systemGeneratedCaseDocuments(systemGeneratedCaseDocuments)
+            .specResponseTimelineDocumentFiles(ResponseDocument.builder()
+                                                   .file(Document.builder()
+                                                       .documentUrl("fake-url")
+                                                       .documentFileName("file-name")
+                                                       .documentBinaryUrl("binary-url")
+                                                       .build()).build())
+            .respondent1SpecDefenceResponseDocument(ResponseDocument.builder().file(Document.builder()
+                                                                                        .documentUrl("fake-url")
+                                                                                        .documentFileName("file-name")
+                                                                                        .documentBinaryUrl("binary-url")
+                                                                                        .build()).build())
+            .build();
+    }
+
+    @Test
+    void shouldGenerateClaimForm_whenNotV1VersionAndIsPinInPostDisabled() {
+        // Given: Case data with docs to stitch, stitching is enabled and isPinInPostEnabled is false
+        ReflectionTestUtils.setField(handler, "stitchEnabled", true);
+        CallbackParams params = callbackParamsOf(caseData, ABOUT_TO_SUBMIT);
+        when(toggleService.isPinInPostEnabled()).thenReturn(false);
```

Comment:
Minor: can use BDDMockito `given` function here.

```suggestion
        given(toggleService.isPinInPostEnabled()).willReturn(false);
```

Make sure to add the relevant import.